calculator_test.go

```diff
@@ -2,23 +2,213 @@ package calculator_test
 
 import (
 	"calculator"
+	"math/rand"
 	"testing"
 )
 
 func TestAdd(t *testing.T) {
+	tcs := []struct {
+		a, b float64
+		rest []float64
+		want float64
+		name string
+	}{
+		{name: "Add two units", a: 0, b: 0, rest: []float64{}, want: 0},
```

Comment:
You might like to try omitting the empty fields and see if that makes these test cases clearer.